src/pages/on-chain-governance/_components/ConfirmVoteDialog.tsx

```diff
@@ -27,33 +37,35 @@ export function ConfirmVoteDialog({
 }: {
   isLoading: boolean;
   setIsLoading: Dispatch<SetStateAction<boolean>>;
-  hasUserSelectedVote: boolean;
-  setHasUserSelectedVote: Dispatch<SetStateAction<boolean>>;
-  isConfirmDetailsClicked: boolean;
-  setIsConfirmDetailsClicked: Dispatch<SetStateAction<boolean>>;
-  isMasterNodeConfirmClicked: boolean;
-  setIsMasterNodeConfirmClicked: Dispatch<SetStateAction<boolean>>;
+  voteStage: VoteStages;
+  setVoteStage: Dispatch<SetStateAction<VoteStages>>;
   setVoteCommand: Dispatch<SetStateAction<string>>;
+  userConfirmedSelectedVote: VoteDecision | undefined;
   setUserConfirmedSelectedVote: Dispatch<SetStateAction<VoteDecision>>;
   proposalId: string;
   isOpen: boolean;
   voteCommand: string;
   onClose: () => void;
 }) {
-  const [masterNodeID, setMasterNodeID] = useState(
-    localStorage.getItem("masternodeID") ?? ""
-  );
+  const { connection } = useNetwork();
+
+  const localStorageMasterNodeId =
+    connection in getLocalStorageItem("masternodeID")
```

Comment:
currently i saw many variation of your casing used:
- masternodeID
- masterNodeID
- rememberMasterNodeId

let choose one and be consistent with it, i'd go with `masternodeId`, rememberMasternodeId`, align with jellyfish too

Reply:
`masterNodeErrorMsg` too